Paco-Server/ear/default/web/js/viz/vizService.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Created by muthuk on 6/28/17.
+ */
+pacoApp.factory('experimentsVizService', ['$http', 'experimentService', function ($http, experimentService) {
+
+    var expObject = '';
+
+    function getExperimentObj(id) {
+
+        var getExp = experimentService.getExperiment(id).then(function (experimentValue) {
+            return experimentValue.data;
+        }, function (error) {
+            console.log(error);
+        });
+
+        expObject = getExp.then(function (experimentObj) {
+            return experimentObj.results[0];
+        }, function (error) {
+            console.log(error);
+        });
+        return expObject;
+    }
+
+    function getEventsResponseDetails(experimentObject) {
+
+        var responseDetails = [];
+        var inputs = experimentObject.groups[0].inputs;
+        inputs.forEach(function (input) {
+            if (input.responseType == "likert") {
+                responseDetails.push({
```

Comment:
Is there a reason to create a new object that is copies of the fields in input instead of just putting the input object in the responseDetails?